### ISSUE TITLE
[engine] Make Excel sink work properly and refactor its code

### ISSUE THREAD

**Issue by miseyu** [reearth/reearth-flow#557]
## Is your feature request related to a problem? Please describe.

Since Excel sink(https://github.com/reearth/reearth-flow/blob/main/engine/runtime/action-sink/src/file/excel.rs) does not seem to be working properly at the moment, You will make the following work as expected, and
https://github.com/reearth/reearth-flow/blob/main/engine/runtime/action-sink/src/file/excel.rs
Please extract functions that are commonly used in Excel
https://github.com/reearth/reearth-flow/tree/main/engine/runtime/common/src
as excel.rs.
Please also perform refactoring at the same time.

**Comment by soma00333** [comment]
I’ll handle this👍 👍

**Comment by soma00333** [comment]
I prepared a sample yaml and ran the workflow, but it produced an unexpected excel file.


### check command
```
$ cargo run --package reearth-flow-cli -- run --workflow ./runtime/examples/plateau/testdata/workflow/example_excel.yml
```

### example_excel.yml
```
# yaml-language-server: $schema=https://raw.githubusercontent.com/reearth/reearth-flow/main/engine/schema/workflow.json
id: 00caad2a-9f7d-4189-b479-153fa9ea36dc
name: "ExcelExample"
entryGraphId: 3e3450c8-2344-4728-afa9-5fdb81eec33a
graphs:
- id: 3e3450c8-2344-4728-afa9-5fdb81eec33a
  name: entry_point
  nodes:
  - id: 90f40a3e-61d3-48e2-a328-e7226c2ad1ae
    name: FeatureCreator
    type: action
    action: FeatureCreator
    with:
      creator: |
        [
          #{
            attribute1: "value1",
            attribute2: "value2"
          },
          #{
            attribute1: "value3",
            attribute2: "value4"
          }
        ]

  - id: f5e66920-24c0-4c70-ae16-6be1ed3b906c
    name: FileWriter
    type: action
    action: FileWriter
    with:
      format: excel
      output: "./output/excel_output.xlsx"

  edges:
  - id: c064cf52-705f-443a-b2de-6795266c540d
    from: 90f40a3e-61d3-48e2-a328-e7226c2ad1ae
    to: f5e66920-24c0-4c70-ae16-6be1ed3b906c
    fromPort: default
    toPort: default
```
### expected output
![image](https://github.com/user-attachments/assets/3237eecd-ab2b-486a-85d7-2776395b7414)

### actual output
![image](https://github.com/user-attachments/assets/d410f5a2-bd2d-4820-8b1e-c94c37552d0b)

**Comment by soma00333** [comment]
@miseyu 
I fixed the bug with [PR#566](https://github.com/reearth/reearth-flow/pull/566).
Can I close this ticket?

**Comment by miseyu** [comment]
@soma00333 
Thank you so much!
I'll close it!
